invenio_app_ils/mail/tasks.py

```diff
@@ -45,6 +61,39 @@ def get_recipients(recipients):
     return recipients
 
 
+def create_email_log_entry(data, log_message):
+    """Create an entry in the email log db."""
+    sender_email = data["recipients"][0]
+    SystemAgent = current_app.config["ILS_PATRON_SYSTEM_AGENT_CLASS"]
+    AnonymousPatron = current_app.config["ILS_PATRON_ANONYMOUS_CLASS"]
+
+    if sender_email in current_app.config["ILS_MAIL_NOTIFY_TEST_RECIPIENTS"]:
```

Comment:
I am not sure why this code here. Can we access to the mail data and access the `patron` field?